### PR TITLE
All morpheus grpc requests interface

### PR DESCRIPTION
**Pull request by AdamBalski**
pls give feedback

### CODE REVIEW

File: proto/morpheus.proto [hunk 1]
```diff
@@ -1,22 +1,43 @@
...
+
+message SendFeedbackRequest {
+    string token = 1;
+    repeated PredictedProductNames goodPredictions = 2;
+    repeated PredictedProductNames badPredictions = 3;
+}
+
 service Morpheus {
     rpc PredictProductNames (PredictProductNamesRequest) returns (PredictProductNamesResponse) {}
+    rpc SendFeedback (SendFeedbackRequest) returns (google.protobuf.Empty) {}
```
Comment (kapiw04) [inline proto/morpheus.proto:41]: Did you consider returning some sort of status code here?

File: proto/morpheus.proto [hunk 2]
```diff
@@ -1,22 +1,43 @@
...
-    string receipt = 1;
-    string token = 2;
+    string token = 1;
+    repeated string products = 2;
+    repeated uint64 productGroups = 3;
 }
 
 message PredictedProductNames {
-    repeated string predictedName = 1;
+    repeated PredictedName predictedName = 1;
```
Comment (kapiw04) [inline proto/morpheus.proto]:
```suggestion
    repeated PredictedName predictedNames = 1;
```

File: proto/morpheus.proto [hunk 3]
```diff
@@ -1,22 +1,43 @@
...
+    string nameOnReceipt = 2;
 }
 
 message PredictProductNamesResponse {
     repeated PredictedProductNames predictedProductNamesList = 1;
```
Comment (kapiw04) [inline proto/morpheus.proto:26]: I think it breaks a naming convention, no?

**Review by kapiw04 — COMMENTED** [review]
LGTM, take a look at some suggestions I gave (mainly naming). Note I only reviewed `morpheus.proto` as from my understanding the rest files are generated.

File: proto/morpheus.proto (hunk 1 above)
Comment (AdamBalski) [reply proto/morpheus.proto:41]: All grpc responses contain a status code (context: https://grpc.io/docs/guides/status-codes/#overview). If you are talking about more elaborate DTOs that contain details, then I am not sure what we'd want to put there, unless you have some ideas.

We can also just go from there and adjust later on when a need arises.

File: proto/morpheus.proto (hunk 2 above)
Comment (AdamBalski) [reply proto/morpheus.proto]: I'll commit that seperately, simultaneously with generated code

File: proto/morpheus.proto (hunk 3 above)
Comment (kapiw04) [reply proto/morpheus.proto:26]: Right, I didn't notice we have a list of lists. 
mb

**Pull request merged**